rpcs3/Emu/Io/GHLtar.cpp

```diff
@@ -0,0 +1,215 @@
+﻿// Guitar Hero Live controller emulator
+
+#include "stdafx.h"
+#include "GHLtar.h"
+#include "Emu/Cell/lv2/sys_usbd.h"
+#include "Input/pad_thread.h"
+#include "Emu/Cell/Modules/cellPad.cpp"
+#include "Emu/RSX/GL/GLHelpers.h"
```

Comment:
what are these even for?
```
#include "Emu/Cell/Modules/cellPad.cpp"
#include "Emu/RSX/GL/GLHelpers.h"
```

Reply:
Accidentally left those in when experimenting how to get pad inputs